packages/bus/src/RetryStrategies/ExponentialBackoffRetryStrategy.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { ExponentialBackoffRetryStrategy } from './ExponentialBackoffRetryStrategy';
+import { instance, mock, reset, verify, when } from 'ts-mockito';
+
+describe('ExponentialBackoffRetryStrategy', () => {
+  let retryStrategy!: ExponentialBackoffRetryStrategy;
+  const mockTest = mock<Test>();
+
+  beforeEach(() => {
+    retryStrategy = new ExponentialBackoffRetryStrategy({ maxDepth: 1 });
+  });
+
+  afterEach(() => {
+    reset(mockTest);
+  });
```

Comment:
Remove redundant brackets
```suggestion
  afterEach(() => reset(mockTest));
```